dev/index.tsx

```diff
@@ -15,24 +15,25 @@
  */
 
 import { createDevApp } from '@backstage/dev-utils';
-import { CatalogClient, catalogApiRef } from '@backstage/plugin-catalog';
-import { githubPullRequestsApiRef, GithubPullRequestsClient } from '../src/api';
 import { plugin } from '../src/plugin';
 
 createDevApp()
   .registerPlugin(plugin)
-  .registerApiFactory({
-    deps: {},
-    factory: () => new GithubPullRequestsClient(),
-    implements: githubPullRequestsApiRef,
-  })
-  .registerApiFactory({
-    deps: {},
-    factory: () =>
-      new CatalogClient({
-        apiOrigin: 'http://localhost:7000',
-        basePath: '/catalog',
-      }),
-    implements: catalogApiRef,
-  })
+  // .registerApiFactory({
+  //   deps: {},
+  //   factory: () => new GithubPullRequestsClient(),
+  //   implements: githubPullRequestsApiRef,
+  // })
+  // .registerApiFactory({
+  //   deps: {},
+  //   factory: () =>
+  //     new CatalogClient({
+  //       // apiOrigin:
+  //       discoveryApi: {
+  //         getBaseUrl: () => Promise.resolve('http://localhost:7000/catalog'),
+  //       },
+  //       // basePath: '',
+  //     }),
+  //   implements: catalogApiRef,
+  // })
```

Comment:
If this doesn't work you can probably just remove the file?

Reply:
I think they might update @backstage/dev-utils to v22 and standalone might start working again so maybe its work to keep it for a while

Reply:
Makes sense.